assignment3/graph.dart

```diff
@@ -0,0 +1,171 @@
+import 'dart:io';
+import 'util.dart';
+import 'functions.dart';
+
+class Graph {
+  int nodeId = 0;
```

Comment:
Don't initialise these values since they are mandatory and needed to create a node.

Reply:
solved using late keyword